### PR TITLE
Add cairo subproject as used in harfbuzz

### PR DESCRIPTION
**Pull request by satmandu**
As per https://github.com/harfbuzz/harfbuzz/blob/main/subprojects/cairo.wrap
and also
https://github.com/harfbuzz/harfbuzz/pull/3501/files

### CODE REVIEW

**Comment by eli-schwartz** [comment]
Like the other PR -- this must be a wrap-file, the WrapDB does not accept git based wraps.

Thirdparty projects are welcome to do whatever they want.

File: subprojects/cairo.wrap [hunk 1]
```diff
@@ -0,0 +1,7 @@
+directory=cairo
```
Comment (eli-schwartz) [inline subprojects/cairo.wrap:1]: This doesn't have a section header at all, hmm.

**Comment by eli-schwartz** [comment]
... and of course CI doesn't even catch the issue, because releases.json doesn't contain the new wrap.

**Comment by satmandu** [comment]
Since https://github.com/mesonbuild/wrapdb/pull/366 has been opened for `cairo`, and seems more comprehensive, I will defer to that PR.

**Pull request closed without merging**